.travis.yml

```diff
@@ -1,20 +1,29 @@
 language: node_js
-node_js:
-- 12.13.1
-script: npm run build:production
-before_deploy: tar -zcvf $TRAVIS_BUILD_DIR/lime-app-$TRAVIS_BRANCH.tar.gz build
-deploy:
-  provider: releases
-  api_key:
-    secure: piqSk7MFmSFohAFFLqH8XLTTi825NIJgOVw+yeeDEjZcRV8prdGSTulvWrLF+94MKR/veIW1DC6F2n5ZOtgzrTRIy+BzRstI6T1Np+h2FO0cg6mBgXnc8zi2G9S5CSR/Y4R5S4JUooTz+Yu2GTldkxJnf5DLl0C62AA2UjQclgWYes10gzO0PlO8nMJe2KkZYrWZwEgBJp+aLFme80mwkPY4F6fq4GL3xYRNRSzPD4SG1Rdj8qe+YyExT/AXsiMUEde86Bqq4U1c8Eaglum7n6Y/SnKvt6I9vpp+eSw2vPGh+ruS2Qb38mAMY1juE7czPQ1Afg4mLenwCoz9YrV/uq5RTtg/+3IpReoygbcJKvB/oqPUNjjONr2xrI/SplkAnan/WC5UHW1sl0hRDLfIsgIE0XhJqmUISSsyxEHIAjExVq1Q7nZp3wbl33bDGz9JM6quMrKgSoLTPSy7UC7x5FzPNxGj9MsAcQtnwIxDXg95iYG6u2mmQqA5BlGPcNl4CT9CIWh0kljRZqrSACN8QSQGjam109pcRdP9LbBJ4QWu6DBqK7MeVcBt1rmZa4cm9cF2ufEkm6V9MZ7k6YS+7tqm8BOYaalu2Sv63mfcGaf3GqYkgoAyUQ4xImQgBdD3e+FoG0P5UUORLPDz7u1BQ9Hv6M2Kdyu2XphiKUHxedk=
-  file: "$TRAVIS_BUILD_DIR/lime-app-$TRAVIS_BRANCH.tar.gz"
-  skip_cleanup: true
-  on:
-    tags: true
-after_deploy:
-- "./.ci/send_pull_request.sh"
-'':
-  "/":
-    ci/deploy:
-      key:
-        secure: hYaoxOvHGU1K2oU72Sz4B1Jk9ORy/UGXVvER1PABTb90Bv9LHIhlirsA2PmWou0fos0cnRrPfvxNVSX3lT0r8lUrNhrO4wtFWp/NJTU1pFINEJfXMRRx2vaUoU53y+xYfh60BylE5mzlQRgEqGDaOJPA29l6aX8z7SzPdJcb9TXf/hkwrA+oQhoI5hv5Z0jcUs3jKY3fguHYrF+Fswb7SnIaPOUd0KRVy1tMMcDU1RABvOmlpO1hXhkfsoc1IR6fnYmIBPwgkNFz/EMhbt2lmNUBS65wgFbchj0zD+F7XRY9lIBbedfHNyPDVGxQoTpS5EekDwymwA3cC+JntrOZvwqu+wnS7AbJX2DYVS2gpWZrwFF+b6jAY002+pmO34qiaG0g696sgNlW4K0C1CqDf8/BKzbOLlvD4D4LTzsmQGas35D4LN/eksKrGEWVJTFdSzfOYMN0K+JWhIF65H8IdSymTb1ULRyDZo5+jEXJnbocB5Ict580Vp++i2tgLdY9KOUnmeGjmUKCOppWVHGnY7TipGcGRFLONVFFhEa5n/bKsOsILSujIcHyokkYyZnGsVVlHfJMioSic091JXzR4qJjiWG9OCBkTejK2tADLRifxTQMYpZzM0xuHjdIuGjWnM7gnm3UQWmLjjjHzKx6HIHaDc1BRFm+7WPcWL6gr/U=
+node_js: stable
+
+install: npm install
+
+stages:
+  - name: storybook
+    if: branch = develop  # Push to devlop or PR to develop
```

Comment:
Comment redundant, please remove (if think otherwise, please correct 'devlop' for 'develop')